src/metabase/api/search.clj

```diff
@@ -457,17 +457,17 @@
   {archived       [:maybe ms/BooleanValue]
    table-db-id    [:maybe ms/PositiveInt]
    created_at     [:maybe ms/NonBlankString]
-   created_by     [:maybe ms/PositiveInt]
+   created_by     [:maybe [:or ms/PositiveInt [:sequential ms/PositiveInt]]]
    last_edited_at [:maybe ms/PositiveInt]
-   last_edited_by [:maybe ms/PositiveInt]
+   last_edited_by [:maybe [:or ms/PositiveInt [:sequential ms/PositiveInt]]]
    verified       [:maybe true?]}
   (query-model-set (search-context {:search-string  q
                                     :archived       archived
                                     :table-db-id    table-db-id
                                     :created-at     created_at
-                                    :created-by     created_by
+                                    :created-by     (u/one-or-many created_by)
```

Comment:
Why isn't this `(set (u/one-or-many created_by))`? Then you wouldn't need to modify `created_at` again in `search-context`. It feels unnecessary to separate the "preparation" of this value in two steps

Reply:
fair, I've made the change in [`b7aa764` (#34302)](https://github.com/metabase/metabase/pull/34302/commits/b7aa76480bd953de5e4ef6fcb01255b6f2b4de0d)